packages/flutter/lib/src/material/typography.dart

```diff
@@ -131,6 +131,12 @@ class TextTheme {
     );
   }
 
+  /// Create a new [TextTheme] where each text style from this object has been
```

Comment:
nit: Creates

Reply:
fixed here and where i copy-pasted it from